ExampleApp/Assets/Scenes/SampleScene.unity

```diff
@@ -5733,8 +5733,8 @@ GameObject:
   - component: {fileID: 1900725528}
   - component: {fileID: 1900725527}
   m_Layer: 0
-  m_Name: Main Camera
-  m_TagString: MainCamera
+  m_Name: Text (TMP)
```

Comment:
why is there still text mesh pro? What are other scene changes about?

Reply:
i think it's possible you had unsaved scene changes, then merged, then saved scene. Since there is a reference to TMP, ill be cheeky and undo them